synthtool/gcp/templates/java_library/.kokoro/requirements.txt

```diff
@@ -434,23 +385,22 @@ six==1.16.0 \
     --hash=sha256:1e61c37477a1626458e36f7b1d82aa5c9b094fa4802892072e49de9c60c4c926 \
     --hash=sha256:8abb2f1d86890a2dfb989f9a77cfcfd3e47c2a354b01111771326f8aa26e0254
     # via
-    #   -r requirements.in
     #   gcp-docuploader
     #   google-auth
     #   python-dateutil
-typing-extensions==4.1.1 \
-    --hash=sha256:1a9462dcc3347a79b1f1c0271fbe79e844580bb598bafa1ed208b94da3cdcd42 \
-    --hash=sha256:21c85e0fe4b9a155d0799430b0ad741cdce7e359660ccbd8b530613e8df88ce2
-    # via -r requirements.in
```

Comment:
Ah looks like this dependency is removed from the newly generated requirements.txt file. Could we verify if it is needed? Looking at the logs of an older kokoro `javadoc` or `stage` job (from Sept) may help us determine if it was installed.

Reply:
Yeah I do see this in the job from ~Oct:
```
Collecting typing-extensions==4.1.1
  Downloading typing_extensions-4.1.1-py3-none-any.whl (26 kB)
Collecting urllib3==1.26.12
  Downloading urllib3-1.26.12-py2.py3-none-any.whl (140 kB)
Collecting zipp==3.6.0
  Downloading zipp-3.6.0-py3-none-any.whl (5.3 kB)
Using legacy 'setup.py install' for pyperclip, since package 'wheel' is not installed.
Installing collected packages: pyasn1, urllib3, six, rsa, pycparser, pyasn1-modules, protobuf, idna, charset-normalizer, certifi, cachetools, requests, googleapis-common-protos, google-auth, cffi, zipp, typing-extensions, jeepney, google-crc32c, google-api-core, cryptography, secretstorage, pyparsing, markupsafe, importlib-metadata, google-resumable-media, google-cloud-core, python-dateutil, pyperclip, pyjwt, packaging, keyring, jinja2, google-cloud-storage, colorlog, click, attrs, gcp-releasetool, gcp-docuploader
    Running setup.py install for pyperclip: started
    Running setup.py install for pyperclip: finished with status 'done'
```

Though I did copy this in from other repos so I need to find earlier versions.

Reply:
I've added in the `typing-extensions` dependency.